pkg/apis/core/validation/validation_test.go

```diff
@@ -5865,6 +5865,149 @@ func TestValidatePodDNSConfig(t *testing.T) {
 	}
 }
 
+func TestValidatePodReadinessGates(t *testing.T) {
+	podReadinessGatesEnabled := utilfeature.DefaultFeatureGate.Enabled("PodReadinessGates")
```

Comment:
nit: can we use the features.PodReadynessGate constant throughout the file, rather than the string?

Reply:
Acked. Fixed in the dependency PR https://github.com/kubernetes/kubernetes/pull/64057